src/plopp/backends/matplotlib/utils.py

```diff
@@ -98,7 +98,7 @@ def is_sphinx_build() -> bool:
 
 def copy_figure(fig: FigureLike, **kwargs) -> FigureLike:
     out = fig.__class__(
-        fig._fig_constructor,
+        fig._view.__class__,
```

Comment:
In case we are not keeping the callable anymore,
should we make sure if the `_view.__class__` is the `View` in `def __init_figure__(self, View, *args, **kwargs):`...?

For example,
```
class FigBase:
  def __init_figure__(self, View, *args, **kwargs):
    self._view = View(*args, **kwargs)
    if not self._view.__class__ is View:
        raise TypeError
```

Cause, you might have something like
```
@dataclasses
class Figure:
  width: int
  height: int

def create_figure(size) -> Figure:
  return Figure(size[0], size[1])

fig = InteractiveFigure(create_figure, size=(10, 10))
```

In this case above, it will create the `fig` as intended, but `copy_figure` will not work.


Or is it too much of worry...? 

```